plugins/follow_me/follow_me.h

```diff
@@ -0,0 +1,194 @@
+     */
+    class Configuration
+    {
+    public:
+        /**
+         * @brief Side to follow target from
+         */
+        enum class FollowDirection {
+            NONE = -1,
+            FRONT_RIGHT = 0, /**< @brief Follow from front right. */
+            BEHIND = 1, /**< @brief Follow from behind. */
+            FRONT = 2, /**< @brief Follow from front. */
+            FRONT_LEFT = 3 /**< @brief Follow from front left. */
+        };
+
+        // Default configuration
+        ///////////////////////////////////////////////////////////
+        /**< @brief Default follow target height. */
+        static constexpr const float DEF_HEIGHT_ABOVE_HOME = 8.0f;
+        /**< @brief Min follow target height. */
+        static constexpr const float MIN_HEIGHT_ABOVE_HOME = 8.0f;
+
+        /**< @brief Distance to follow target from. */
+        static constexpr const float DEF_DIST_WRT_FT = 8.0f;
+        /**< @brief Min distance to follow target from. */
+        static constexpr const float MIN_DIST_WRT_FT = 1.0f;
+
+        /**< @brief Side to follow target from. */
+        static constexpr const FollowDirection DEF_FOLLOW_DIR = FollowDirection::BEHIND;
+
+        /**< @brief Default dynamic filtering algorithm responsiveness to target movement. */
+        static constexpr const float DEF_DYN_FLT_ALG_RSP = 0.5f;
+        /**< @brief Min dynamic filtering algorithm responsiveness to target movement. */
+        static constexpr const float MIN_DYN_FLT_ALG_RSP = 0.0f;
+
+        /**< @brief Max dynamic filtering algorithm responsiveness to target movement. */
+        static constexpr const float MAX_DYN_FLT_ALG_RSP = 1.0f;
+        ///////////////////////////////////////////////////////////
+
+        // FollowMe Configuration Accessors.
+        /**
+         * @brief Gets minimum follow target height, in meters.
+         * @return Minimum follow target height.
+         * @sa set_min_height_m(float)
+         */
+        float min_height_m() const;
+
+        /**
+         * @brief Gets distance to follow target from, in meters.
+         * @return Current distance to follow target from, in meters.
+         * @sa set_follow_target_dist_m(float)
+         */
+        float follow_target_dist_m() const;
+
+        /**
+         * @brief Gets Side to follow target from.
+         * @return %FollowDirection representing Side to follow target from.
+         * @sa set_follow_dir(FollowDirection), to_str(FollowDirection)
+         */
+        FollowDirection follow_dir() const;
+
+        /**
+         * @brief String form of %FollowDirection
+         * @param dir enum %FollowDirection
+         * @return std::string representation of enum %FollowDirection
+         * @sa follow_dir(), set_follow_dir(FollowDirection)
+         */
+        static std::string to_str(FollowDirection dir);
+
+        /**
+         * @brief Gets current dynamic filtering algorithm responsiveness value.
+         * @return Current responsiveness value
```

Comment:
What is the unit of this?

Reply:
No unit for this: Check for `NAV_FT_RS ` in the table in https://docs.px4.io/en/advanced_config/parameter_reference.html#follow-target

Reply:
Ok, then describe what the range and meaning is, I guess.